docs/Getting Started/Deploying to AWS/index.md

```diff
@@ -0,0 +1,135 @@
+## Deploying to AWS
+
+### Deploying your JavaEE App on EC2
```

Comment:
All headers should be updated to fit the new top header

Reply:
fixed